kiosk/stylelint.config.js

```diff
@@ -1,22 +1,24 @@
 module.exports = {
-  extends: ["stylelint-config-standard"],
+  extends: ['stylelint-config-standard'],
   rules: {
-    "at-rule-no-unknown": [
+    'at-rule-no-unknown': [
       true,
       {
         ignoreAtRules: [
-          "tailwind",
-          "apply",
-          "variants",
-          "responsive",
-          "screen",
-          "layer",
-          "mixin",
-          "include",
+          'tailwind',
+          'apply',
+          'variants',
+          'responsive',
+          'screen',
+          'layer',
+          'mixin',
+          'include',
         ],
       },
     ],
-    "declaration-block-trailing-semicolon": null,
-    "no-descending-specificity": null,
+    'declaration-block-trailing-semicolon': null,
+    'no-descending-specificity': null,
+    'value-keyword-case': null,
```

Comment:
The additions to this file are not in sync with the other stylelint configurations we have in other projects. Please run `pre-pr-updates.ps1` or at least `compare-synced-files.ps1` and fix the files that should be in sync.

Reply:
What is the reason of these additions to the stylelint config? What are they improving?

Reply:
they make sure all the configs are the same between apps (tv, kiosk, billbaord and web)

Reply:
leaving it in as its not a problem on any other channels